src/lib/cost_model/cost.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+namespace opossum {
+
+/**
+ * Cost that an AbstractCostModel assigns to an Operator/LQP node. The unit of the Cost is left to the Cost model and could be,
+ * e.g., "Estimated Runtime" or "Estimated Memory Usage".
+ */
+using Cost = float;
```

Comment:
I wonder if the name could somehow reflect that "cost" is nondimensional. Until I saw this file, I thought it was always the runtime.